native/src/components/CityNotCooperatingFooter.tsx

```diff
@@ -38,6 +38,8 @@ const CityNotCooperatingFooter = ({
   if (!buildConfig().featureFlags.cityNotCooperating) {
     return null
   }
+  // This footer is only displayed in the Integreat configuration, and that has the CityNotCooperatingIcon
```

Comment:
Please remove this comment, can easily become outdated, I think best solution here would be to add to the check above (line 38), that also the icon needs to be present.